### DISCUSSION TITLE
Add Total to record List block

### DISCUSSION THREAD

**Discussion by alrikvidal** [cortezaproject/corteza-server#2044] [Ideas]
Hello,

I propose the idea of adding a TOTAL row at the bottom of the page, which would be dynamically calculated based on the filters applied in the Record List block.

**Comment by Fajfa** [comment]
https://docs.cortezaproject.org/corteza-docs/2024.9/integrator-guide/compose-configuration/page-blocks.html#page-block-rl-count

Do you mean this kind of total? If so it already exists.

**Reply by alrikvidal** [reply]
No, I already use that feature. I mean summing the column, like shown on the screen :

![image](https://github.com/user-attachments/assets/4aa02ff5-93ab-4337-a3e4-011abadbbc80)

**Reply by Fajfa** [reply]
Understood, to do something similar you can use a Metric block.

Feel free to open an issue describing your suggestions.

**Reply by alrikvidal** [reply]
I use Metric, But if i apply filter on my Record List block, metric block don't apply this filter
